gradle/libs.versions.toml

```diff
@@ -131,6 +131,11 @@ timber = { module = "com.jakewharton.timber:timber", version.ref = "timber" }
 test-harness = { module = "com.google.accompanist:accompanist-testharness", version.ref = "testHarness" }
 work-runtime-ktx = { module = "androidx.work:work-runtime-ktx", version.ref = "workRuntimeKtx" }
 
+chucker-debug =  "com.github.chuckerteam.chucker:library:4.0.0"
```

Comment:
Please check the above definitions as references which follow the format
```
[versions]
...
<library_version> = 4.0.0
...
[libraries]
...
name = {module = "<module name>", version.ref="<library_version>"}
```
And are also defined in alphabetical order

Reply:
Ok will make the necessary changes